target/linux/x86/64/config-5.15

```diff
@@ -487,10 +491,11 @@ CONFIG_X86_INTEL_LPSS=y
 CONFIG_X86_INTEL_PSTATE=y
 CONFIG_X86_MINIMUM_CPU_FAMILY=64
 # CONFIG_X86_PCC_CPUFREQ is not set
-CONFIG_X86_PKG_TEMP_THERMAL=y
+# CONFIG_X86_PKG_TEMP_THERMAL is not set
```

Comment:
Why do you deactivate `CONFIG_X86_PKG_TEMP_THERMAL` ?

Reply:
I understood `CONFIG_INTEL_HFI_THERMAL=y` to supply similar functionality making this setting redundant.  The model desktop distro I used to compare does not have this option enabled.  I can toggle it back if you feel it is needed.